[WIP] Configuration Files for Enhanced OpenFL Documentation

manuelhsantana opened the pull request:
This PR serves as a complement to the recently updated OpenFL documentation [[https://github.com/securefederatedai/openfl/pull/1003]](url) by providing the necessary configuration files. These files are essential for the proper integration and functionality of the new OpenFL API section introduced in the previous PR.

Key additions in this PR include:

- Configuration Files: Addition of configuration files that are required for the new OpenFL API section to be fully operational. These files ensure that the updated documentation and API section are correctly displayed and accessible to the users.

- Integration Support: The configuration files included in this PR support the integration of the updated docstrings and the newly added indexes and tables, ensuring a seamless user experience.

- Compliance with Standards: The configuration files adhere to the same flake8 formatting standards as the updated documentation files, maintaining consistency across the platform.

This PR is crucial for the implementation of the comprehensive documentation update and is designed to work in tandem with the previous PR to enhance the usability and understanding of the OpenFL platform.

manuelhsantana committed:
Added functionality files for docstring API documentation

Signed-off-by: Manuel Santana <manuel.santana.castolo@intel.com>

manuelhsantana committed:
Added functionality files for docstring API documentation

Signed-off-by: Manuel Santana <manuel.santana.castolo@intel.com>

manuelhsantana committed:
Fix federated rendering for docstrings

Signed-off-by: Manuel Santana <manuel.santana.castolo@intel.com>

manuelhsantana committed:
Fix federated rendering for docstrings

Signed-off-by: Manuel Santana <manuel.santana.castolo@intel.com>

MasterSkepticista commented on openfl/federated/__init__.py:
```diff
@@ -14,3 +14,9 @@
 if pkgutil.find_loader('torch'):
     from .task import PyTorchTaskRunner, FederatedModel  # NOQA
     from .data import PyTorchDataLoader, FederatedDataSet  # NOQA
+
+__all__ = [
+    'Plan',
+    'TaskRunner',
+    'DataLoader',
+]
```
Can we remove this change? After #998 is merged, this will become a source of merge conflict. Also #998 removes this construct.

psfoley replied:
This was added at my request. \_\_all\_\_ is necessary for the API docs to render correctly. Without this, entire pages were missing in the prior PR. Maybe it's possible to modify the configuration to avoid this though.

kminhta committed:
[Fix] Update requirement.txt to install latest torch version in torch workspaces (#1000)

* remove unused pt_cnn.py from torch_cnn_histology

Signed-off-by: kta-intel <kevin.ta@intel.com>

* update validate to validate_task in unet_kvasir plan.yaml

Signed-off-by: kta-intel <kevin.ta@intel.com>

* update package versions

Signed-off-by: kta-intel <kevin.ta@intel.com>

---------

Signed-off-by: kta-intel <kevin.ta@intel.com>
Signed-off-by: Manuel Santana <manuel.santana.castolo@intel.com>

psfoley committed:
Merge branch 'develop' into docstring_API_func

porteratzo committed:
option to autodoc all imported files

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

porteratzo committed:
option to autodoc all imported files

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

porteratzo committed:
rerun tests

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

porteratzo committed:
rerun tests

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

MasterSkepticista committed:
Enable TLS by default (#1012)

* Enable TLS by default

Signed-off-by: Shah, Karan <karan.shah@intel.com>

* Spurious package removal in dockerfile

Signed-off-by: Shah, Karan <karan.shah@intel.com>

* Fetch fqdn via openfl util func

Signed-off-by: Shah, Karan <karan.shah@intel.com>

* [WIP] Set localhost fqdn for github actions

Signed-off-by: Shah, Karan <karan.shah@intel.com>

* formatting break from PR #1008 fixed

Signed-off-by: Shah, Karan <karan.shah@intel.com>

* Set localhost fqdn for impacted tests

Signed-off-by: Shah, Karan <karan.shah@intel.com>

* localhost fqdn for dockerization test

Signed-off-by: Shah, Karan <karan.shah@intel.com>

---------

Signed-off-by: Shah, Karan <karan.shah@intel.com>

dependabot[bot] committed:
build(deps): update pyzmq requirement from <=26.0.3 to <=26.1.0 (#1020)

Updates the requirements on [pyzmq](https://github.com/zeromq/pyzmq) to permit the latest version.
- [Release notes](https://github.com/zeromq/pyzmq/releases)
- [Commits](https://github.com/zeromq/pyzmq/compare/v2.0.7...v26.1.0)

---
updated-dependencies:
- dependency-name: pyzmq
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump torch from 1.13.1 to 2.2.0 in /tests/github/experimental/workspace/testcase_private_attributes_initialization_with_both_options (#1019)

* build(deps): bump torch

Bumps [torch](https://github.com/pytorch/pytorch) from 1.13.1 to 2.2.0.
- [Release notes](https://github.com/pytorch/pytorch/releases)
- [Changelog](https://github.com/pytorch/pytorch/blob/main/RELEASE.md)
- [Commits](https://github.com/pytorch/pytorch/compare/v1.13.1...v2.2.0)

---
updated-dependencies:
- dependency-name: torch
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

* upgrade torch and torchvision

Signed-off-by: kta-intel <kevin.ta@intel.com>

---------

Signed-off-by: dependabot[bot] <support@github.com>
Signed-off-by: kta-intel <kevin.ta@intel.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: kta-intel <kevin.ta@intel.com>

dependabot[bot] committed:
build(deps): bump pytest-asyncio from 0.23.7 to 0.23.8 (#1006)

Bumps [pytest-asyncio](https://github.com/pytest-dev/pytest-asyncio) from 0.23.7 to 0.23.8.
- [Release notes](https://github.com/pytest-dev/pytest-asyncio/releases)
- [Commits](https://github.com/pytest-dev/pytest-asyncio/compare/v0.23.7...v0.23.8)

---
updated-dependencies:
- dependency-name: pytest-asyncio
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

porteratzo committed:
rerun tests

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

porteratzo committed:
rerun tests

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

dependabot[bot] committed:
build(deps): bump dynaconf from 3.2.5 to 3.2.6 (#1022)

Bumps [dynaconf](https://github.com/dynaconf/dynaconf) from 3.2.5 to 3.2.6.
- [Release notes](https://github.com/dynaconf/dynaconf/releases)
- [Changelog](https://github.com/dynaconf/dynaconf/blob/3.2.6/CHANGELOG.md)
- [Commits](https://github.com/dynaconf/dynaconf/compare/3.2.5...3.2.6)

---
updated-dependencies:
- dependency-name: dynaconf
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): update grpcio-tools requirement (#1007)

Updates the requirements on [grpcio-tools](https://github.com/grpc/grpc) to permit the latest version.
- [Release notes](https://github.com/grpc/grpc/releases)
- [Changelog](https://github.com/grpc/grpc/blob/master/doc/grpc_release_schedule.md)
- [Commits](https://github.com/grpc/grpc/compare/v1.56.2...v1.65.1)

---
updated-dependencies:
- dependency-name: grpcio-tools
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

kminhta committed:
Update torch from 1.13.1 to 2.3.1 (and dependent packages) in existing tutorials and workspaces (#1024)

* upgrade torch and torchvision in github tests

Signed-off-by: kta-intel <kevin.ta@intel.com>

* update torch==2.3.1 and torchvision==0.18.1 in experimental workspace

Signed-off-by: kta-intel <kevin.ta@intel.com>

* bump torch==2.3.1 torchvision==0.18.1 in interactive_api tutorials

Signed-off-by: kta-intel <kevin.ta@intel.com>

* update torch=2.3.1 torchvision 0.18.1

Signed-off-by: kta-intel <kevin.ta@intel.com>

* bump torch=2.3.1 torchvision=0.18.1 for workflow api tutorials

Signed-off-by: kta-intel <kevin.ta@intel.com>

---------

Signed-off-by: kta-intel <kevin.ta@intel.com>

kminhta committed:
bump torch==2.3.1, torchvision==0.18.1 (#1025)

Signed-off-by: kta-intel <kevin.ta@intel.com>

kminhta committed:
Resolve coverity scan 441135 `bad use of null-like value` in openfl/experimental/runtime/local_runtime.py (#1026)

* resolve coverity scan: bad use of null-like value

Signed-off-by: kta-intel <kevin.ta@intel.com>

* style reformat

Signed-off-by: kta-intel <kevin.ta@intel.com>

---------

Signed-off-by: kta-intel <kevin.ta@intel.com>

manuelhsantana committed:
Update Docstrings api for documentation (issue #893) (#1003)

* build(deps): bump pytest-mock from 3.10.0 to 3.14.0

Bumps [pytest-mock](https://github.com/pytest-dev/pytest-mock) from 3.10.0 to 3.14.0.
- [Release notes](https://github.com/pytest-dev/pytest-mock/releases)
- [Changelog](https://github.com/pytest-dev/pytest-mock/blob/main/CHANGELOG.rst)
- [Commits](https://github.com/pytest-dev/pytest-mock/compare/v3.10.0...v3.14.0)

---
updated-dependencies:
- dependency-name: pytest-mock
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* build(deps-dev): update grpcio-tools requirement (#986)

Updates the requirements on [grpcio-tools](https://github.com/grpc/grpc) to permit the latest version.
- [Release notes](https://github.com/grpc/grpc/releases)
- [Changelog](https://github.com/grpc/grpc/blob/master/doc/grpc_release_schedule.md)
- [Commits](https://github.com/grpc/grpc/compare/v1.56.2...v1.64.1)

---
updated-dependencies:
- dependency-name: grpcio-tools
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* build(deps): bump pytest from 8.2.0 to 8.2.2 (#987)

Bumps [pytest](https://github.com/pytest-dev/pytest) from 8.2.0 to 8.2.2.
- [Release notes](https://github.com/pytest-dev/pytest/releases)
- [Changelog](https://github.com/pytest-dev/pytest/blob/main/CHANGELOG.rst)
- [Commits](https://github.com/pytest-dev/pytest/compare/8.2.0...8.2.2)

---
updated-dependencies:
- dependency-name: pytest
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* fix the installtion doc

Signed-off-by: myan <myan@redhat.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* correct all

Signed-off-by: myan <myan@redhat.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated sphinx config for API doc and added Index

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated aggregator api documentation

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated assigner api documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated collaborator api documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated director api documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated director api documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated envoy api documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Fix typos and remove comments on rst file

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added cryptography module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added TensorDB module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added native module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added Experimental module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added Federated module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added interface module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added pipelines module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added plugins module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added protocols module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added transport module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Added utilities module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated component module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated cryptography module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated databases module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated native module docstrings for API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated modules configuration for docstrings API documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update conflict files for merge

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Updated different files for docstrings API documentation

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Fixed problem of path for documentation

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Moved the location of Indices and tables

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update aggregator docstring for fix lint warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update aggregator docstring for fix lint warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update assigner functions docstring for fix lint warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update component module functions docstrings for fix lint warnings

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update component imports docstring for fix lint warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update cryptography functions docstring for fix lint warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update databases functions docstring for fix lint warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update databases functions docstring for fix lint warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update databases functions docstring for fix lint warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update experimental functions docstring for fix lint warnings

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update transport functions docstring for fix lint warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update tensor_db file for avoid lint warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update federated functions docstring for fix lint flake8 warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update interface functions docstring for fix lint flake8 warnings

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update native functions docstring for fix lint flake8 warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update pipeline functions docstring for fix lint flake8 warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update plugin functions docstring for fix lint flake8 warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update protocols functions docstring for fix lint flake8 warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update utilities functions docstring for fix lint flake8 warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update minor files docstring for fix lint flake8 warnings

Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Convert relative imports to absolute

Signed-off-by: Shah, Karan <karan.shah@intel.com>
Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update different functions docstring for fix lint warnings

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Files updated to comply with flake8 format

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Files updated to comply with flake8 format

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Files updated to comply with more flake8 format

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Revert __init__.py files to fix doc rendering

Signed-off-by: Patrick Foley <psfoley@gmail.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Removed all functionality changes

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Removed changes on imports

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* [Fix] Update requirement.txt to install latest torch version in torch workspaces (#1000)

* remove unused pt_cnn.py from torch_cnn_histology

Signed-off-by: kta-intel <kevin.ta@intel.com>

* update validate to validate_task in unet_kvasir plan.yaml

Signed-off-by: kta-intel <kevin.ta@intel.com>

* update package versions

Signed-off-by: kta-intel <kevin.ta@intel.com>

---------

Signed-off-by: kta-intel <kevin.ta@intel.com>
Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* lint fixes

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* lint fixes

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* Update assigner.py

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* build(deps): bump pytest from 8.2.2 to 8.3.2 (#1014)

Bumps [pytest](https://github.com/pytest-dev/pytest) from 8.2.2 to 8.3.2.
- [Release notes](https://github.com/pytest-dev/pytest/releases)
- [Changelog](https://github.com/pytest-dev/pytest/blob/main/CHANGELOG.rst)
- [Commits](https://github.com/pytest-dev/pytest/compare/8.2.2...8.3.2)

---
updated-dependencies:
- dependency-name: pytest
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* revert again

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* undo lint eddit

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* undo lint edits

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* undo reqs

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* rerun tests

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* lintfix

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

* rerun tests

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>

---------

Signed-off-by: porteratzo <omar.alfonso.montoya@hotmail.com>
Signed-off-by: myan <myan@redhat.com>
Signed-off-by: manuelhsantana <manuel.santana.castolo@intel.com>
Signed-off-by: Shah, Karan <karan.shah@intel.com>
Signed-off-by: Patrick Foley <psfoley@gmail.com>
Signed-off-by: kta-intel <kevin.ta@intel.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: myan <myan@redhat.com>
Co-authored-by: Shah, Karan <karan.shah@intel.com>
Co-authored-by: Patrick Foley <psfoley@gmail.com>
Co-authored-by: Kevin Ta <116312994+kta-intel@users.noreply.github.com>
Co-authored-by: porteratzo <omar.alfonso.montoya@hotmail.com>
Co-authored-by: porteratzo <44075849+porteratzo@users.noreply.github.com>

MasterSkepticista committed:
Merge branch 'docstring_API_func' of https://github.com/securefederatedai/openfl into docstring_API_func

MasterSkepticista committed:
Pytest FQDN fix

Signed-off-by: Shah, Karan <karan.shah@intel.com>

MasterSkepticista committed:
Run formatter

Signed-off-by: Shah, Karan <karan.shah@intel.com>
